packages/core/test/DirectoryLoader.test.ts

```diff
@@ -654,18 +645,6 @@ describe('DirectoryLoader', () => {
 			expect(nodeWithIcon.description.icon).toBeUndefined();
 		});
 
-		it('should skip node if included in excludeNodes', () => {
-			const loader = new CustomDirectoryLoader(directory, ['CUSTOM.node1']);
```

Comment:
removed this because now exclusion doesn't work for custom nodes. If someone is adding custom nodes to their instance, and doesn't want it to be used, they can simply remove the node.